src/card.tsx

```diff
@@ -407,6 +407,7 @@ interface MediaCardBaseProps {
     subtitle?: string;
     subtitleLinesMax?: number;
     description?: string;
+    icon?: React.ReactElement
```

Comment:
In figma it was called Asset, you may need to make this switch from icon to Asset
@FernandoRRosa

Reply:
yes, better call "asset" will be the new name for all props that contains assets. We want to make this breaking change in the next major and rename `icon` to `asset` to be more generic →  https://github.com/Telefonica/mistica-web/issues/626

In my opinion, we have to embrace the new nomenclature and use `asset`

Reply:
hmmm... I think we should still use icon, as we do in other Card components. I agree with Yayo that we have a pending rename for this prop, but we should do that rename in all the cards at the same time. That rename will be done in a future mistica major release, not now